tox.ini

```diff
@@ -16,7 +16,7 @@ skip_install = true
 recreate=True
 commands =
     pip --version
-    lowest: pip install flask<2 markupsafe~=2.0.0
+    lowest: pip install flask<2 sqlalchemy<2 markupsafe~=2.0.0
```

Comment:
flask<2 results in flask-sqlalchemy<3, which requires sqlalchemy<2

Reply:
We can adjust the lowest version we are testing for.  Could just follow what flask-sqlalchemy is testing: https://github.com/pallets-eco/flask-sqlalchemy/blob/main/tox.ini